mooncake-store/src/master_service.cpp

```diff
@@ -318,6 +318,21 @@ ErrorCode MasterService::Remove(const std::string& key) {
     return ErrorCode::OK;
 }
 
+ErrorCode MasterService::RemoveAll() {
+    for (auto& shard : metadata_shards_) {
+        std::unique_lock<std::mutex> lock(shard.mutex);
+
+        // Clear all metadata
+        for (auto it = shard.metadata.begin(); it != shard.metadata.end(); ) {
+            // Remove metadata will trigger replica deletion.
+            it = shard.metadata.erase(it);
```

Comment:
Would it be possible to invoke clear directly? This might be a more efficient approach.

Reply:
```cpp
        for (auto& shard : metadata_shards_) {
            std::unique_lock<std::mutex> lock(shard.mutex);
            shard.metadata.clear();
        }
 ```